lib/cachedir.js

```diff
@@ -1,38 +1,70 @@
-var fs = require('fs-ext'),
-  log = require('sslog'),
-  path = require('path'),
-  request = require('request'),
-  crypto = require('crypto'),
-  mkdirp = require('mkdirp'),
-  sprintf = require('util').format;
-
-function md5(string, callback) {
+var fs = require('fs');
+var log = require('sslog');
+var path = require('path');
+var request = require('request');
+var crypto = require('crypto');
+var mkdirp = require('mkdirp');
+var sprintf = require('util').format;
+
+/**
+ * Util function used to generate cache filenames.
+ */
+function md5(str, callback) {
   if (typeof callback !== 'function') callback = function() {};
```

Comment:
I know this wasn't your code, but could we just return early and not go through hashing the string here when callback isn't a function (instead of assigning it to an empty function)?